WORKING_GROUPS.md

```diff
@@ -0,0 +1,239 @@
+# OpenJS Core Working Groups
+
+OpenJS Core Working Groups (WG) are autonomous projects created by the
+[Cross Project Council (CPC)][].
+
+Working Groups can be formed at any time but must be ratified by the CPC.
+Once formed the work defined in the Working Group charter is the
+responsibility of the WG rather than the CPC.
+
+It is important that Working Groups are not formed pre-maturely. Working
+Groups are not formed to *begin* a set of tasks but instead are formed
+once that work is already underway and the contributors
+think it would benefit from being done as an autonomous project.
+
+If the work defined in a Working Group's charter is complete, the charter
+should be revoked.
+
+A Working Group's charter can be revoked either by consent of the Working
+Group's members or by a CPC vote. Once revoked, any future work that arises
+becomes the responsibility of the CPC.
```

Comment:
On revocation, should Github projects related to the WG's work be archived unless WG members request otherwise and the CPC agrees?

It seems that doing so by default might help keep the organization project list manageable.

Reply:
I agree, this should at least enforce that the project list stays manageable. If the CPC is very dependent on the WG's projects, the transfer of responsibility will likely already be well known during the time of revocation anyway. 👍

Reply:
I say let's not be prescriptive and see if this needs to be optimized. It has been working in the Node.js space for some time as is. Thanks.